src/components/common/alert/alert.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { VFC } from 'react';
+import { Alert as AlertNB, HStack, VStack, Text } from 'native-base';
+
+import styles from './alert.style';
+
+export const Alert: VFC<Props> = ({ status, text, isShown }) => {
+  return (
+    <AlertNB
+      w="75%"
+      status={status}
+      style={[styles.wrapper, isShown ? styles.isShown : styles.isHidden]}
+    >
+      <VStack space={2} flexShrink={1} w="100%">
```

Comment:
Usually in refactoring those numbers can be called "magic numbers". Let's move them to the separate file and export them from there as constants, i.e.
`import {spaceSize} from ./constants.tsx .... space={spaceSize}`

Reply:
same for colors and better to place all colors from the app to one general constants file and import them from there

Reply:
Put magic numbers and colors to constants